doc/classes/Curve2D.xml

```diff
@@ -107,11 +107,12 @@
 			<param index="0" name="offset" type="float" default="0.0" />
 			<param index="1" name="cubic" type="bool" default="false" />
 			<description>
-				Similar to [method sample_baked], but returns [Transform2D] that includes a rotation along the curve. Returns empty transform if length of the curve is [code]0[/code].
+				Similar to [method sample_baked], but returns [Transform2D] that includes a rotation along the curve, with [member Transform2D.origin] as the point position, [member Transform2D.x] as the sideways vector, and [member Transform2D.y] as the forward vector. Returns empty transform if length of the curve is [code]0[/code].
+				[b]Note:[/b] To use this to transform nodes you need to do, this will point the node with up along the curve:
```

Comment:
Something is wrong with this sentence. I can't understand what it's trying to say. Is a word missing, or something?

Reply:
Will try clarify, it is supposed to say that the up direction of the node will be in the tangent direction of the curve

Reply:
```
[codeblock]
# This will rotate and position the node with the up direction pointing along the curve.
var baked = curve.sample_baked_with_rotation(offset)
position = baked.get_origin()
rotation = baked.get_rotation()
[/codeblock]
```

Is this clearer?

Reply:
Yes, but what's the first half of the current message about? "To use this to transform nodes you need to do".

Reply:
Oh I removed that was unsure how to phrase it, just using the example